libs/wingsdk/src/util/util.ts

```diff
@@ -35,6 +69,32 @@ export class Util {
     return new Promise((resolve) => setTimeout(resolve, delay.seconds * 1000));
   }
 
+  /**
+   * Waits until predicate is true or return false
```

Comment:
Please document what the return value means (`@returns your description`)

Reply:
Use capital `Y` in the return doc:

```
@returns Your description
```

(@Chriscbr we need @return documentation to start with a capital case).

(maybe add to the linter?)

Reply:
Yes, we need a style guide...